src/app/cluster/list/external-cluster/template.html

```diff
@@ -69,47 +69,52 @@
             *matCellDef="let element"
             [attr.id]="'km-clusters-' + element.name">
           <div fxLayoutAlign=" center"
-               fxLayoutGap="8px">{{element.name}}
+               fxLayoutGap="8px">
+            <span>{{element.name}}</span>
+            <span *ngIf="(!element?.labels || element?.labels['is-imported'] !== 'true')"
+                  class="km-label-primary">Imported</span>
           </div>
         </td>
       </ng-container>
 
-      <ng-container matColumnDef="labels">
+      <ng-container [matColumnDef]="Column.Provider">
         <th mat-header-cell
             *matHeaderCellDef
-            class="km-header-cell p-30">Labels
+            class="km-header-cell p-20">Provider
         </th>
         <td mat-cell
             *matCellDef="let element">
-          <km-labels [labels]="element.labels"
-                     [limit]="2"></km-labels>
+          <span [ngClass]="getProvider(element.cloud) === Provider.Custom ?
+              'km-provider-logo km-provider-logo-custom-external-cluster' : 'km-provider-logo km-provider-logo-'+getProvider(element.cloud)"></span>
         </td>
       </ng-container>
 
-      <ng-container matColumnDef="provider">
+
+      <ng-container [matColumnDef]="Column.Region">
         <th mat-header-cell
             *matHeaderCellDef
-            class="km-header-cell p-20">Provider
+            class="km-header-cell p-20">Region
         </th>
         <td mat-cell
             *matCellDef="let element">
-          <span [ngClass]="getProvider(element.cloud) === Provider.Custom ?
-              'km-provider-logo km-provider-logo-custom-external-cluster' : 'km-provider-logo km-provider-logo-'+getProvider(element.cloud)"></span>
+          <ng-container *ngIf="element.cloud">
+            {{getRegion(element.cloud)}}
+          </ng-container>
         </td>
       </ng-container>
 
-      <ng-container matColumnDef="created">
+      <ng-container [matColumnDef]="Column.Created">
```

Comment:
maybe we need to change it to `Column.Imported`

Reply:
Actually label was correct shown in UI but from Code perspective it was created. FYI here `created` was due to the reason because external cluster can be created via wizard or can be imported by dialog but for both still show Import as heading but actually underlying it is `creationTimeStamp` managed by backend but you were right for better CONTEXT it was better to rename column **binding** name